### PR TITLE
docs: Fix missing critical steps in how-to-hotplug-memory-arm64.md

### PR DESCRIPTION
**Pull request by singhwang**
The key steps in how-to-hotplug-memory-arm64.md are missing, resulting in the kata qemu pod not being created successfully.

Fixes: #6105
Signed-off-by: SinghWang <wangxin_0611@126.com>

### CODE REVIEW

**Comment by singhwang** [comment]
I often encounter this problem, but the corresponding configuration is not found in the official documents. Today in the kata WeChat exchange group, I found that someone has encountered the same problem again, so I initiated this pr, hoping It can help the kata official community to improve the documentation.

File: docs/how-to/how-to-hotplug-memory-arm64.md [hunk 1]
```diff
@@ -15,6 +15,19 @@ $ sudo .ci/aarch64/install_rom_aarch64.sh
...
+## Config KATA QEMU
+
+After executing the above script, two files will be generated under the directory /usr/share/kata-containers/ by default, namely kata-flash0.img and kata-flash1.img. Next we need to change the configuration file of kata qemu, which is in /opt/kata/share/defaults/kata-containers/configuration-qemu.toml by default, specify in the configuration file to use the UEFI ROM installed above. Please refer to the following configuration:
+
+```
+[hypervisor.qemu]
+
+# -pflash can add image file to VM. The arguments of it should be in format
+# of ["/path/to/flash0.img", "/path/to/flash1.img"]
+# pflashes = []
```
Comment (liubin) [inline docs/how-to/how-to-hotplug-memory-arm64.md:27]: Please remove this line.

File: docs/how-to/how-to-hotplug-memory-arm64.md [hunk 2]
```diff
@@ -15,6 +15,19 @@ $ sudo .ci/aarch64/install_rom_aarch64.sh
 $ popd
 ```
 
+## Config KATA QEMU
+
+After executing the above script, two files will be generated under the directory /usr/share/kata-containers/ by default, namely kata-flash0.img and kata-flash1.img. Next we need to change the configuration file of kata qemu, which is in /opt/kata/share/defaults/kata-containers/configuration-qemu.toml by default, specify in the configuration file to use the UEFI ROM installed above. Please refer to the following configuration:
```
Comment (liubin) [inline docs/how-to/how-to-hotplug-memory-arm64.md:20]: Kata Containers has spelling checks for markdown, if the result shows some errors, you can add backticks to quote the words, especially for filenames or directories.

File: docs/how-to/how-to-hotplug-memory-arm64.md (hunk 1 above)
Comment (singhwang) [reply docs/how-to/how-to-hotplug-memory-arm64.md:27]: Do you want me to remove all these comments?

File: docs/how-to/how-to-hotplug-memory-arm64.md (hunk 2 above)
Comment (singhwang) [reply docs/how-to/how-to-hotplug-memory-arm64.md:20]: Thanks for the tip, I'll adjust it based on follow-up results.

File: docs/how-to/how-to-hotplug-memory-arm64.md (hunk 1 above)
Comment (liubin) [reply docs/how-to/how-to-hotplug-memory-arm64.md:27]: No, only line `# pflashes = []`, this is the original line in the configuration file, you have changed it to `pflashes = ["/usr/share/kata-containers/kata-flash0.img", "/usr/share/kata-containers/kata-flash1.img"]`

Comment (singhwang) [reply docs/how-to/how-to-hotplug-memory-arm64.md]: ok!

**Review by cmaf — APPROVED** [review]
thanks @singhwang

**Comment by singhwang** [comment]
@liubin @cmaf  There was a spelling issue before, which I have fixed. At the same time, I checked again, this time using the official spell checking tool `$GOPATH/src/github.com/kata-containers/tests/cmd/check-spelling/kata-spell-check.sh `prompted in the error message, Verification has been confirmed.
```
./kata-spell-check.sh check /data/go/src/github.com/kata-containers/kata-containers/docs/how-to/how-to-hotplug-memory-arm64.md
INFO: Spell checking file '/data/go/src/github.com/kata-containers/kata-containers/docs/how-to/how-to-hotplug-memory-arm64.md'
INFO: Spell check successful for file: '/data/go/src/github.com/kata-containers/kata-containers/docs/how-to/how-to-hotplug-memory-arm64.md'
```

**Review by GabyCT — APPROVED** [review]

**Comment by singhwang** [comment]
There should be nothing to modify now. Can it be merged please?

**Comment by liubin** [comment]
> There should be nothing to modify now. Can it be merged please?

This will be merged after all the tests have passed, please wait for a moment.

**Comment by singhwang** [comment]
@liubin You may need to merge manually. Currently, an error is reported that is not a required type. I still don't know how to solve this error.
Add backport label / check-issues (pull_request) :
https://github.com/kata-containers/kata-containers/actions/runs/3964567165/jobs/6804166664

**Comment by singhwang** [comment]
@liubin On the main branch do I still need to resubmit the PR for this issue?

**Comment by liubin** [comment]
@singhwang yes, if the patch is not in main

**Pull request merged**